File: Android/java/doraemonkit/src/main/java/com/didichuxing/doraemonkit/kit/performance/PerformanceCloseListener.java
package com.didichuxing.doraemonkit.kit.performance;


/**
 * ================================================
 * 作    者：jint（金台）
 * 版    本：1.0
 * 创建日期：2019-10-11-20:39
 * 描    述：性能监控关闭回调接口 系统模式下专用
 * 修订历史：
 * ================================================
 */
public interface PerformanceCloseListener {

    /**
     * @param performanceType
     */
    void onClose(int performanceType);
}

File: Android/kotlin/doraemonkit/src/main/java/com/didichuxing/doraemonkit/kit/performance/manager/PerformanceCloseListener.kt
package com.didichuxing.doraemonkit.kit.performance.manager

/**
 *
 * Desc:性能监控关闭回调接口 系统模式下专用
 * <p>
 * Date: 2020-06-09
 * Company:
 * Updater:
 * Update Time:
 * Update Comments:
 *
 * Author: pengyushan
 */
interface PerformanceCloseListener {
    /**
     * @param performanceType
     */
    fun onClose(performanceType: Int)
}